media-libs/avisynthplus/avisynthplus-3.7.3.recipe

```diff
@@ -57,7 +58,6 @@ INSTALL()
 {
 	make -C build install
 
-	prepareInstalledDevelLib libavisynth
```

Comment:
Why is it removed?

Reply:
See the discussion when the recipe was first added, https://github.com/haikuports/haikuports/pull/5614#issuecomment-762809293

prepareInstalledDevelLib breaks client programs because most of them interface with AviSynth through dlopen.  Most prominently:
[FFmpeg](http://git.videolan.org/?p=ffmpeg.git;a=blob;f=libavformat/avisynth.c;h=b426ac343e67e6357c39cbd15b326be1d53add68;hb=2cd09fec9f8d3b16d03e762dec6001cc0b792659#l35)
[x264](https://code.videolan.org/videolan/x264/-/blob/master/input/avs.c#L29)
[avs2yuv](https://github.com/DJATOM/avs2yuv/blob/master/avs2yuv.c#L20)

Reply:
Maybe this deserves a comment in the recipe, otherwise we will keep adding it at every update. I suspect doing it this way results in some warnings from haikuporter strict mode?

Reply:
prepareInstalledDevelLib is correct, because any client programs not using dlopen has its build broken.
Please add the desired symlink in $libDir manually.

Reply:
BTW I see no recipe at Haikuports which depends on lib:libavisynth. How do you test?

Reply:
Testing was done with already existing source builds of FFmpeg and avs2yuv.  avs2yuv doesn't exist as a recipe (it probably should, but that's a separate issue), and FFmpeg can only enable AviSynth support if `--enable-gpl` is used; if there was a decision to have an alternate, ffmpeg_gpl package, then it could go in there.

Of the existing ports, x264 is tethered to commit bfc87b7a330f75f5c9a21e56081e4b20344f139e, [which enables AviSynth+ on Haiku](https://code.videolan.org/videolan/x264/-/commit/bfc87b7a330f75f5c9a21e56081e4b20344f139e).  x264 auto-enables the avs input module since it carries its own copy of the headers with it and doesn't need a _devel package to enable it.

Reply:
OK I understand. For x264, it means one could add a `SUPPLEMENTS="x264$secondaryArchSuffix"` or `SUPPLEMENTS_bin="x264$secondaryArchSuffix"` in the avisynthplus recipe.